packaging/kickstart/kickstart-bootc.ks.template

```diff
@@ -0,0 +1,68 @@
+lang en_US.UTF-8
+keyboard us
+timezone UTC
+text
+reboot
+
+# Partition the disk with hardware-specific boot and swap partitions, adding an
+# LVM volume that contains a 10GB+ system root. The remainder of the volume will
+# be used by the CSI driver for storing data.
+zerombr
+clearpart --all --initlabel
+# Create boot and swap partitions as required by the current hardware platform
+reqpart --add-boot
+# Add an LVM volume group and allocate a system root logical volume
```

Comment:
No minimum size. I'd like to avoid complicating instructions further. What we have here is a sensible start, I think. Users can then change the values if necessary.